content/en/docs/concepts/configuration/overview.md

```diff
@@ -37,6 +37,23 @@ to others, please don't hesitate to file an issue or submit a PR.
 
 - Put object descriptions in annotations, to allow better introspection.
 
+{{< note >}}
+There is is a breaking change introduced in the [YAML 1.2](https://yaml.org/spec/1.2.0/#id2602744)
```

Comment:
```suggestion
There is a breaking change introduced in the [YAML 1.2](https://yaml.org/spec/1.2.0/#id2602744)
```
Small typo!

Reply:
Thank you @stormqueen1990 
Can't believe I rephrased the paragraph and repeated the original typo :)